scripts/x-data-grid-pro.exports.json

```diff
@@ -343,6 +344,8 @@
   { "name": "GridRowProps", "kind": "Interface" },
   { "name": "GridRowScrollEndParams", "kind": "Interface" },
   { "name": "GridRowSelectionCheckboxParams", "kind": "Interface" },
+  { "name": "GridRowsInternalCache", "kind": "Interface" },
```

Comment:
If we can avoid to export those `GridRowsInternalCache`, `GridRowsInternalCacheState` and `GridRowInternalCacheValue`